invenio_rdm_records/contrib/thesis/custom_fields.py

```diff
@@ -25,15 +25,19 @@
 THESIS_CUSTOM_FIELDS_UI = {
     "section": _("Thesis"),
     "fields": [
-        dict(
-            field="thesis:university",
-            ui_widget="Input",
-            props=dict(
-                label="Awarding university",
-                icon="building",
-                description="University name",
-                placeholder="",
-            ),
-        )
+        {
+            "field": "thesis:university",
+            "ui_widget": "Thesis",
```

Comment:
Had to implement a new widget so the UI is consistent (compared to other custom fields e.g. `journal`, `imprint`)